packages/hyperion-flowlet/src/Flowlet.ts

```diff
@@ -22,4 +22,8 @@ export class Flowlet<T extends {} = {}> {
   fork(name: string): Flowlet<T> {
     return new Flowlet<T>(name, this);
   }
-}
\ No newline at end of file
+}
```

Comment:
You cannot change this package based on the needs of autologging. This type is already defined in hyperion-autologging package.